backend/main.py

```diff
@@ -0,0 +1,99 @@
+import sqlite3, uuid, json
+from bottle import Bottle, run, request, response
+
+app = Bottle()
+
+# DB mit SQLite
+con = sqlite3.connect("users.db")
+cur = con.cursor()
+
+def init():
+    createDB()
+
+def createDB():
```

Comment:
Would it make sense to already add some type of database migration logic? Like https://pypi.org/project/yoyo-migrations

Reply:
depending on how extensive the backend is, this could be helpful

Reply:
See #13